tiledb/sm/filter/test/unit_double_delta_pipeline.cc

```diff
@@ -117,7 +117,7 @@ static bool expect_overflow(
     }
     for (uint64_t i = 1; i < values.size(); i++) {
       std::optional<int64_t> delta =
-          checked_arithmetic<int64_t>::sub_signed(values[i], values[i - 1]);
+          checked_arithmetic<uint64_t>::sub_signed(values[i], values[i - 1]);
```

Comment:
`values` is `std::span<const uint64_t>` here, this was implicitly converting to `int64_t` before subtracting which led to the wrong result.